packages/search/test/content/index.test.ts

```diff
@@ -263,8 +263,9 @@ describe("Content Search Service", () => {
         "/search",
         {
           hubApiUrl: "https://hubqa.arcgis.com",
+          portal: "https://qaext.arcgis.com/sharing/rest",
           authentication,
-          isPortal: false,
```

Comment:
I'm guessing this was `false` b/c previously `isPortal` was required, but it's now optional since the code was just checking if it was falsy.

Reply:
I think this is still ok for functionality.